resilience4j-spring/src/main/java/io/github/resilience4j/circuitbreaker/configure/CircuitBreakerConfiguration.java

```diff
@@ -64,13 +64,13 @@ public CircuitBreakerAspect circuitBreakerAspect(CircuitBreakerRegistry circuitB
 
 
 	@Bean
-	@Conditional(value = {RxJava2OnClasspathCondition.class})
+	@Conditional(value = {RxJava2OnClasspathCondition.class, AspectJOnClasspathCondition.class})
```

Comment:
I guess we need to do the same for the rest (Bulkhead ext..ect) plus we need to do the same on the circuit breaker aspect bean injection

Reply:
Which class do you mean?

Reply:
RxJava2RateLimiterAspectExt,ReactorRateLimiterAspectExt beans injection in RateLimiterConfigutation , same for Retry and Bulkhead plus in CircuitBreakerConifguration , add the same condition over CircuitBreakerAspect bean injection

Reply:
done